tests/cnf/core/network/cni/tests/sysctl/e2e.go

```
@@ -92,9 +93,78 @@ var _ = Describe("CNF Sysctl", Ordered, Label(tsparams.LabelSysctlTestCases), Co
+					validMacVlanInterfaces[0].Name)
+
+				By("Define and create NAD without sysctl config")
+				createStaticIpamNad(
+					tsparams.NetworkWithoutSysctlMutation, validMacVlanInterfaces[0].Name)
+
+				By("Create server and redirect pod")
+				createServerPodWithInit(defineDualServerNetCfg(), tsparams.SrvDualInitCMD)
+				createRedirectPodWithInit(defineDualRedirectNetCfg(), tsparams.RdrDualInitCMD)
+
+				By("Create client pod connected to both NADs")
+				runningClientPod := createClientPodWithInit(defineDualClientNetCfg(), tsparams.ClientDualInitCMD)
```

golangci-lint (2.13.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)